packages/unplugin-typia/tests/fixtures/__snapshots__/vite/random.js

```
@@ -1,46 +1,120 @@
+var hasRequired_randomString;
+function require_randomString() {
+  if (hasRequired_randomString) return _randomString;
+  hasRequired_randomString = 1;
+  Object.defineProperty(_randomString, "__esModule", { value: true });
+  _randomString._randomString = void 0;
+  const _randomInteger_1 = /* @__PURE__ */ require_randomInteger();
+  const _randomString$1 = (props) => {
+    var _a;
+    const length = (0, _randomInteger_1._randomInteger)({
+      type: "integer",
+      minimum: (_a = props.minLength) !== null && _a !== void 0 ? _a : 0,
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)